qiskit/transpiler/preset_passmanagers/default.py

```diff
@@ -71,15 +71,17 @@ def default_pass_manager(basis_gates, coupling_map, initial_layout, seed_transpi
     # Change CX directions
     pass_manager.append(CXDirection(coupling_map))
 
-    # Unroll to the basis
-    pass_manager.append(Unroller(['u1', 'u2', 'u3', 'id', 'cx']))
-
-    # Simplify single qubit gates and CXs
-    simplification_passes = [Optimize1qGates(), CXCancellation(), RemoveResetInZeroState()]
-
+    # simplifications and optimizations
+    simplification_passes = [RemoveResetInZeroState(), CXCancellation()]
+    if set(basis_gates).issubset(set(['u1', 'u2', 'u3', 'id', 'cx', 'U', 'CX'])):
+        # Simplify single qubit gates
+        simplification_passes += [Optimize1qGates()]
```

Comment:
why U C

Reply:
remove?

Reply:
because `Optimize1qGates()` also works on `U` and `CX`.

Reply:
it really should not. These are ground truths for the matrix. We really should not be using them anywhere in Qiskit.

Reply:
Ups. I was wrong. `Optimize1qGates` does not work on `U` or `CX`. Fixed in 653995d0 
```
In:             qr = QuantumRegister(1, 'qr') 
   ...:         circuit = QuantumCircuit(qr) 
   ...:         circuit.u_base(1,2,3,qr) 
   ...:         circuit.u_base(2,2,1,qr)  
   ...:         dag = circuit_to_dag(circuit) 
   ...:         expected = QuantumCircuit(qr) 
   ...:  
   ...:         pass_ = Optimize1qGates() 
   ...:         after = pass_.run(dag) 
   ...:  
   ...:         print(dag_to_circuit(after))                                                                                                           
         ┌──────────┐┌──────────┐
qr_0: |0>┤ U(1,2,3) ├┤ U(2,2,1) ├
         └──────────┘└──────────┘
```